app/pages/corrections/RemoveCountryCorrectionPage.scala

```diff
@@ -45,4 +45,13 @@ case class RemoveCountryCorrectionPage(periodIndex: Index) extends QuestionPage[
         case _ => controllers.corrections.routes.CorrectPreviousReturnController.onPageLoad(CheckMode, answers.period)
       }
     }
+
+  override def navigateInCheckThirdLoopMode(answers: UserAnswers): Call =
+    answers.get(DeriveNumberOfCorrections(periodIndex)) match {
+      case Some(n) if n > 0 => controllers.corrections.routes.VatCorrectionsListController.onPageLoad(CheckThirdLoopMode, answers.period, periodIndex)
```

Comment:
Can we rename n and x in here to more useful names

Reply:
done